packages/@webex/plugin-meetings/src/locus-info/selfUtils.ts

```diff
@@ -443,7 +447,10 @@ SelfUtils.localAudioUnmuteRequestedByServer = (oldSelf: any = {}, changedSelf: a
     );
   }
 
-  return changedSelf.localAudioUnmuteRequested && !oldSelf.localAudioUnmuteRequested;
+  return (
+    changedSelf.localAudioUnmuteRequested &&
+    changedSelf.localAudioUnmuteRequestedTimeStamp > oldSelf.localAudioUnmuteRequestedTimeStamp
```

Comment:
see my below comment. we don't want to compare ISO strings this way. instead let's convert them to ms with `Date.parse`. that way the `<` will work as expected since it will be comparing numbers vs. numbers instead of strings vs. strings